src/upload-course-content.js

```diff
@@ -143,7 +143,8 @@ module.exports = class UploadCourseContent {
 				DueDate: topic.dueDate || null,
 				Url: `${orgUnit.Path}${fileName}`,
 				IsHidden: isHidden,
-				IsLocked: false
+				IsLocked: false,
+				IsExempt: !topic.isRequired
```

Comment:
Do we need to worry about a topic being hidden and required?

Reply:
Maybe? Given we have control of the content, not so much.
I'll open an issue since it's not high priority.